collections.md

```diff
@@ -287,6 +318,31 @@ The `avg` method returns the [average value](https://en.wikipedia.org/wiki/Avera
 
     // 2
 
+<a name="method-before"></a>
+#### `before()` {.collection-method}
+
+The `before` method is the opposite of the [`after`](#method-after) method. It returns the item before the given item. It returns `null` if the given item is not found or is the first item:
```

Comment:
I didn't cover the strict comparison and closure here since they are mentioned in the `after` method.